libraries/provision/ansible/playbooks/tasks/stop-sg-accel.yml

```diff
@@ -1,11 +1,11 @@
 # Stop service
 - name: SG ACCEL | stop sg_accel service
   service: name=sg_accel state=stopped
-  when: ansible_distribution == "CentOS" and ansible_distribution_major_version == "7"
+  when: (ansible_distribution == "CentOS" and ansible_distribution_major_version == "7") or (ansible_distribution == "Ubuntu")
 
 - name: SG ACCEL | stop sg_accel for CentOS 6
```

Comment:
Replace Centos to generic name or you can remove it or add other platforms too which added in the condition